docs/upgrade.html

```diff
@@ -89,6 +89,27 @@ <h5><a id="upgrade_400_notable" href="#upgrade_400_notable">Notable changes in 4
                 </li>
                 <li><b>Broker</b>
                     <ul>
+                        <li>The new Consumer Group Rebalance Protocol is now a production ready feature, enabled by default on the broker.
```

Comment:
nit: Should we add `(KIP-848)` after `Consumer Group Rebalance Protocol` and link the KIP?